src/libs/SidebarUtils.ts

```diff
@@ -352,6 +352,11 @@ function getOptionData({
                         : ` ${Localize.translate(preferredLocale, 'workspace.invite.from')}`;
                 result.alternateText += `${preposition} ${roomName}`;
             }
+            if (lastActionName === CONST.REPORT.ACTIONS.TYPE.ROOM_CHANGE_LOG.UPDATE_ROOM_DESCRIPTION) {
+                result.alternateText = `${lastActorDisplayName} ${Localize.translate(preferredLocale, 'roomChangeLog.updateRoomDescription')} ${
+                    lastActionOriginalMessage?.description
+                }`.trim();
+            }
```

Comment:
We always display `set the room description to:` here and it caused [this bug ](https://github.com/Expensify/App/issues/45522).
More detail in this comment:  https://github.com/Expensify/App/issues/45522#issuecomment-2232596441